tests/cases/unittests/session.ts

```diff
@@ -0,0 +1,459 @@
+/// <reference path="..\..\..\src\harness\harness.ts" />
+
+module ts.server {
+	let lastWrittenToHost: string,
+	mockHost: ServerHost = {
+		args: [],
+		newLine: '\n',
+		useCaseSensitiveFileNames: true,
+		write: (s) => lastWrittenToHost = s,
+		readFile: () => void 0,
+		writeFile: () => void 0,
+		resolvePath: () => void 0,
+		fileExists: () => false,
+		directoryExists: () => false,
+		createDirectory: () => void 0,
+		getExecutingFilePath: () => void 0,
+		getCurrentDirectory: () => void 0,
+		readDirectory: () => void 0,
+		exit: () => void 0

```

Comment:
Got it. In my own code I've adopted `() => void 0` as an amazingly short noop function that is visually less ambiguous than `() => {}` (which for those less informed about lambdas looks like returning an empty object) and _very_ similar to its type definition: `() => void`. The preference simply bled through here.